js/tests/play-with-variables_test.json

```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "description": "escapeFromDelimiters is declared and includes a double-quote",
+    "code": "if (typeof escapeFromDelimiters === 'undefined') {\n  throw Error(\n    `You didn't even define the variable... we've been through this already !`,\n  )\n}\n\nif (!escapeFromDelimiters.includes('\"')) {\n  throw Error('escapeFromDelimiters must include a double-quote\"')\n}"
+  },
+  {
+    "description": "escapeFromDelimiters includes a single-quote",
+    "code": "if (!escapeFromDelimiters.includes(\"'\")) {\n  throw Error(\"escapeFromDelimiters must include a single-quote'\")\n}"
+  },
+  {
+    "description": "escapeFromDelimiters includes a backtick",
+    "code": "if (!escapeFromDelimiters.includes('`')) {\n  throw Error('escapeFromDelimiters must include a backtick `')\n}"
+  },
+  {
+    "description": "escapeTheEscape includes a backslash",
+    "code": "if (!new TextEncoder().encode(escapeTheEscape).includes(92)) {\n  throw Error('escapeTheEscape must includes a backslash')\n}"
+  },
+  {
+    "description": "The value of power must have changed",
+    "code": "let power = 'under 9000'\n\n// Your code\n\nequal(love, 'levelMax')"
```

Comment:
https://github.com/01-edu/public/pull/2667#discussion_r1698546657

also about this test i have a question here please @nprimo